lib/DDG/Spice/Cve_summary.pm

```diff
@@ -0,0 +1,42 @@
+package DDG::Spice::Cve_summary;
+# ABSTRACT: Write an abstract here
+# Start at https://duck.co/duckduckhack/spice_overview if you are new
+# to instant answer development
+
+use DDG::Spice;
+
+spice is_cached => 1;
+# API url
+spice to => 'http://cve.circl.lu/api/cve/$1';
+spice wrap_jsonp_callback => 1;
+
+# Metadata.  See https://duck.co/duckduckhack/metadata for help in filling out this section.
+name "Cve_summary";
+source "http://cve.circl.lu";
+#icon_url "";
+description "Displays a short summary of a Common Vulnerabilities and Exposures (CVE).";
+primary_example_queries "cve CVE-2016-0815";
+#secondary_example_queries "optional -- demonstrate any additional triggers";
+# Uncomment and complete: https://duck.co/duckduckhack/metadata#category
+category "reference";
+# Uncomment and complete: https://duck.co/duckduckhack/metadata#topics
+# topics "";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/Cve_summary.pm";
+attribution github => ["GitHubAccount", "MoriTanosuke"],
+            twitter => "carsten_r";
+
+# Triggers
+triggers any => "cve";
+
+# Handle statement
+handle remainder => sub {
```

Comment:
let's `handle query_lc` and then make sure that the query is of the form `CVE-\d{4}-\d+`. You'll probably want to also strip out a leading `cve` if it exists.

Judging by the autocomplete the majority of searches are just `cve-####-####`